modules/delegates.js

```diff
@@ -97,6 +99,47 @@ class Delegates {
 	}
 }
 
+/**
+ * Caches delegate list for last 2 rounds.
+ *
+ * @private
+ * @param {number} round - Round Number
+ * @param {array} delegatesList - Delegate list
+ * @returns {Void}
+ */
+__private.updateDelegateListCache = function(round, delegatesList) {
```

Comment:
Unit test missing for `updateDelegateListCache` function

Reply:
@lsilvs Created https://github.com/LiskHQ/lisk/issues/2593